modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/rocksdb/TxStateRocksDbStorage.java

```diff
@@ -146,7 +146,7 @@ public void close() {
             reverse(resources);
             closeAll(resources);
         } catch (Exception e) {
-            throw new IgniteInternalException("Failed to stop transaction state storage of the table: " + id, e);
+            throw new TxStateStorageException("Failed to stop transaction state storage of the table: " + id, e);
```

Comment:
Optional, I would do something like this: `throw new TxStateStorageException("Failed to stop transaction state storage of the table: {}", e, id);`